src/OpenTelemetry.SemanticConventions/scripts/semantic-conventions/generate.ps1

```diff
@@ -3,9 +3,9 @@ $SCRIPT_DIR=$PSScriptRoot
 $ROOT_DIR="${SCRIPT_DIR}/../../"
 
 # freeze the spec & generator tools versions to make SemanticAttributes generation reproducible
-$SPEC_VERSION="v1.13.0"
+$SPEC_VERSION="v1.17.0"
 $SCHEMA_URL="https://opentelemetry.io/schemas/$SPEC_VERSION"
-$GENERATOR_VERSION="0.14.0"
+$GENERATOR_VERSION="0.15.0"
```

Comment:
I see also 0.15.1 version. Does it fix mentioned issue?

Reply:
I believe I tried both but I'll double check.

Reply:
Yea I get the same error with 0.14, 0.15, and 0.15.1; but I updated to 0.15.1 anyway.